cupy/testing/helper.py

```diff
@@ -49,7 +49,11 @@ def _check_cupy_numpy_error(self, cupy_error, cupy_tb, numpy_error,
         self.fail('Only numpy raises error\n\n' + numpy_tb)
     elif numpy_error is None:
         self.fail('Only cupy raises error\n\n' + cupy_tb)
-    elif type(cupy_error) is not type(numpy_error):
+    elif not isinstance(cupy_error, type(numpy_error)):
+        # CuPy errors should always be more explicit than NumPy errors, i.e.
```

Comment:
Strictly speaking this is not exactly correct, because they can be the same errors.
Can you fix?

Reply:
Thanks, I fixed that one.